andela/ah-frontend-odin
# #161340140 Enable social login using facebook & google

**What does this PR do?**
This PR creates an interface for the user to log into the system using google and facebook. 

**Description of Task to be completed?**
Enable users to Login using their facebook and google accounts

**How should this be manually tested?**

- Start by cloning the repository to your local machine
           git clone https://github.com/andela/ah-frontend-odin
- Change to the project directory
           cd ah-frontend-odin/
-Install the package dependencies using the command below
           npm install
- Start the app by running 
           npm start
- if a user wishes to Login using google, then they should click on the button labelled "google" and "facebook" if they wish to Login using Facebook

**Any background context you want to provide?**
None

**What are the relevant pivotal tracker stories?**
[Finishes [#161340140](https://www.pivotaltracker.com/story/show/161340140)]

<img width="1227" alt="screen shot 2018-12-13 at 12 37 03" src="https://user-images.githubusercontent.com/42246312/49929592-e480c900-fed3-11e8-9149-3c3702937fda.png">


<img width="1279" alt="screen shot 2018-12-06 at 14 52 39" src="https://user-images.githubusercontent.com/42246312/49582600-a67d2580-f966-11e8-86dd-3effa232b1c0.png">

<img width="1100" alt="screen shot 2018-12-13 at 09 28 23" src="https://user-images.githubusercontent.com/42246312/49919966-8646ec80-feb9-11e8-91c9-bf6e08a5732a.png">
